packages/fiori/src/themes/Page.css

```diff
@@ -29,24 +29,57 @@
 }
 
 .ui5-page-footer-root {
+    width: 100%;
+    box-sizing: border-box;
     position: absolute;
     bottom: 0;
     left: 0;
     z-index: 2;
     width: 100%;
 }
 
-:host([disable-scrolling]) .ui5-page-content-root {
-    overflow: hidden;
-}
-
 :host([floating-footer]) .ui5-page-footer-root {
-    left: 0.5rem;
-    right: 0.5rem;
-    width: auto;
     opacity: 1;
     bottom: 0.5rem;
 }
+/*** Responsive paddings ***/
```

Comment:
Last thing,  should we also handle the case when a **ui5-bar is used in the footer, when it is not floating**, but regular one - similar to the header styles `:host([media-range="S"]) ::slotted([ui5-bar][slot="header"])`:

```css
:host([media-range="S"]:not([floating-footer])) ::slotted([ui5-bar][slot="footer"] {
```

Reply:
When the bar is not floating, it should always has a width of 100% from the page. (No paddings at all)